config/allowed-licenses.json

```diff
@@ -20,6 +20,9 @@
     },
     {
       "moduleLicense": "Bouncy Castle Licence"
+    },
+    {
+      "moduleLicense": "COMMON DEVELOPMENT AND DISTRIBUTION LICENSE (CDDL) Version 1.0"
     }
```

Comment:
This license is needed for `javax.annotation:javax.annotation-api` which is pulled in by `io.grpc:grpc-kotlin-stub`